lib/plaid/products/deposit_switch.rb

```diff
@@ -0,0 +1,100 @@
+module Plaid
+  # Public: Class used to call the Deposit Switch product.
+  class DepositSwitch < BaseProduct
+    # Public: Get Deposit Switch information.
+    #
+    # Does a POST /deposit_switch/get call which fetches deposit switch
+    # associated with a deposit switch id.
+    #
+    # deposit_switch_id - deposit_switch_id of a deposit switch to fetch
+    #                     deposit switch for
```

Comment:
nit: `s/deposit_switch_id of a deposit switch to fetch deposit switch for/deposit_switch_id to fetch deposit switch for`

Reply:
⚡️